### DISCUSSION TITLE
eccentricity

### DISCUSSION THREAD

**Discussion by akraskovSMDB** [neuropoly/axondeepseg#635] [Help needed] [answered]
Hi

what is the meaning of eccentricity metric if "-a circle" option was used?

Thank you

**Comment by hermancollin** [comment, marked as answer]
Hi @akraskovSMDB,

The axon shape (circle or ellipse) does not affect the eccentricity computation. This is the exact definition of the eccentricity value:

> Eccentricity of the ellipse that has the same second-moments as the region. The eccentricity is the ratio of the focal distance (distance between focal points) over the major axis length. The value is in the interval [0, 1). When it is 0, the ellipse becomes a circle.

I will confirm this with the team, but  as far as I know, changing the axon shape affects 3 things:
- axon diameter
- myelin thickness
- g-ratio
